src/shared/test/settings-test.js

```diff
@@ -2,33 +2,128 @@
 
 var settings = require('../settings');
 
-function createConfigElement(obj) {
-  var el = document.createElement('script');
-  el.type = 'application/json';
-  el.textContent = JSON.stringify(obj);
-  el.classList.add('js-hypothesis-config');
-  el.classList.add('js-settings-test');
-  return el;
-}
-
-function removeJSONScriptTags() {
-  var elements = document.querySelectorAll('.js-settings-test');
-  for (var i=0; i < elements.length; i++) {
-    elements[i].parentNode.removeChild(elements[i]);
-  }
-}
-
 describe('settings', function () {
-  afterEach(removeJSONScriptTags);
+  describe('#jsonConfigsFrom', function() {
+    var jsonConfigsFrom = settings.jsonConfigsFrom;
 
-  it('reads config from .js-hypothesis-config <script> tags', function () {
-    document.body.appendChild(createConfigElement({key:'value'}));
-    assert.deepEqual(settings(document), {key:'value'});
-  });
+    function appendJSHypothesisConfig(document_, jsonString) {
+      var el = document_.createElement('script');
+      el.type = 'application/json';
+      el.textContent = jsonString;
+      el.classList.add('js-hypothesis-config');
+      el.classList.add('js-settings-test');
+      document_.body.appendChild(el);
+    }
+
+    afterEach('remove js-hypothesis-config tags', function() {
+      var elements = document.querySelectorAll('.js-settings-test');
+      for (var i=0; i < elements.length; i++) {
+        elements[i].parentNode.removeChild(elements[i]);
```

Comment:
You can shorten this to `elements[i].remove()`. Some older code uses `removeChild()` because `Element#remove` was not supported in older versions of PhantomJS.